pkg/monitor/write_job_run_data.go

```diff
@@ -68,7 +68,12 @@ func computeDisruptionData(eventIntervals monitorapi.Intervals) *BackendDisrupti
 	}
 
 	allBackendLocators := sets.String{}
-	allDisruptionEventsIntervals := eventIntervals.Filter(monitorapi.IsDisruptionEvent)
+	allDisruptionEventsIntervals := eventIntervals.Filter(
+		monitorapi.And(
+			monitorapi.IsDisruptionEvent,
+			monitorapi.IsErrorEvent, // ignore Warning events, we use these for disruption we don't actually think was from the cluster under test (i.e. DNS)
+		),
+	)
```

Comment:
I'm not sure this was needed, I did a trial run first linked in the comments where I just moved the intervals to warning from error. The disruption data file was all 0. I don't know where it's filtering though.

Reply:
Looks like it is filtered here: https://github.com/openshift/origin/blob/b827d56d6ec689900714209b41b23a29dbad3188/pkg/monitor/monitorapi/disruption.go#L12

Reply:
Awesome thanks Ken! I'm going to leave it double filtered for now I think, seems safe to do.